src/_resources/fact-sheet-merchant-category-codes-in-mobility.md

```diff
@@ -0,0 +1,26 @@
+---
+date: "2023-08-01T17:00:00-07:00"
+title: "Fact Sheet: Merchant Category Codes in Mobility"
```

Comment:
`title` should be Title case

```suggestion
title: "Fact sheet: Merchant category codes in mobility"
```

Reply:
Do you mean sentence case? (Basically make the casing consistent to all the other titles)

Reply:
It seems like "Merchant Category Codes" should have proper-noun capitalization, at least from what I see in the resource content.

I'm just gonna go ahead and apply your suggestion because I agree that the casing should be consistent across all content. Someone can tell me to change "Merchant category codes" if it is incorrect. Process-wise, I think it'd be helpful if the submitter could apply the right casing so we don't have to worry about it.

Reply:
Yeah we sort of touched on this on the call -- we have to be the last line of defense and use our best judgement, since most people will never care about style guides.